README.md

```diff
@@ -75,7 +75,7 @@ If you want, you can apply some filters on the states name, for example :
 app.config(/* @ngInject */ (atInternetUiRouterPluginProvider) => {
   // replace all occurrences of "foo" by "bar" in states name
   atInternetUiRouterPluginProvider
-    .addStateNameFilter(stateName => stateName.replace(/foo/g, 'bar'));
+    .addStateNameFilter((stateName) => stateName.replace(/foo/g, 'bar'));
```

Comment:
The rules have changed ?

Reply:
Yes!

Related to the bump of `eslint-config-airbnb-base@^14.0.0`.